doc/source/tutorial/stats/outliers.md

```diff
@@ -0,0 +1,521 @@
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
++++ {"tags": ["jupyterlite_sphinx_strip"]}
+
+```{eval-rst}
+.. notebooklite:: outliers.md
+   :new_tab: True
+```
+
+(outliers)=
+
++++
+
+# Trimming and Winsorization Transition Guide
+
+An outlier is an observation that differs substantially from other observations in a sample. There are many references available about identifying outliers, whether to make adjustments for outliers, and if so, what adjustments to make for outliers. This guide does not make any recommendations on these topics. Instead, the focus is how to perform two of the most common procedures:
+
+- trimming: removing outliers from the dataset, and
+- winsorization: replacing outliers with a more central value,
+
+using tools available in SciPy, with an emphasis on transitiong from use of legacy functions to the preferred approach.
+
+# Motivation
+In the past, SciPy has offered several "convenience functions" that combine trimming with computation of a statistic. Consider, for instance, `scipy.stats.trim_mean`.
+
+```{code-cell} ipython3
+import numpy as np
+from scipy import stats
+np.set_printoptions(linewidth=120)
+x = np.arange(20.)
+stats.trim_mean(x, proportiontocut=0.1)
+```
+
+Without carefully reading the documentation, it is not immediately obvious how the operation was performed. In this case, we would have gotten the same result whether `proportiontocut` refered to the proportion to cut from each tail or the total amount of data to remove.
+
+```{code-cell} ipython3
+# Remove 2/20 observations total before taking the `mean`, one in each tail
+np.mean(x[1:-1])
+```
+
+```{code-cell} ipython3
+# Remove 2/20 observations from *each* tail before taking the `mean`
+np.mean(x[2:-2])
+```
+
+Likewise, when the proportion would not result in an integral number of points being removed, how many points are removed? For example, in an array with 22 elements, is the number to remove rounded up or down?
+
+```{code-cell} ipython3
+x = np.arange(22.)
+# Remove `floor(0.1 * 22)` (2) observations from each tail
+np.mean(x[2:-2])
+```
+
+```{code-cell} ipython3
+# Remove `ceil(0.1 * 22)` (2) observations from each tail
+np.mean(x[3:-3])
+```
+
+What if instead of removing a proportion, we want to remove data below and above certain threshholds. It turns out that we would need to use `scipy.stats.tmean` or (the similarly-named, but curiously relegated to a separate namespace) `scipy.stats.mstats.trimmed_mean`, instead.
+
+```{code-cell} ipython3
+stats.tmean(x, limits=(2, 19), inclusive=(True, True))
+```
+
+```{code-cell} ipython3
+stats.mstats.trimmed_mean(x, limits=(2, 19), relative=False, inclusive=(True, True))
+```
+
+What if we want to remove data outside a "confidence interval" estimate. Do any of these functions - or perhaps `scipy.stats.trim` followed by `np.mean` - offer anything to facilitate that?
```

Comment:
Does "outside a 'confidence interval' estimate make sense? What I am trying to refer to is outside specified percentiles of the data, but I don't want to use those terms because it is so similar to trimming a specified percentage of the data. (Apparently, the difference is so subtle, that the two are often treated interchangably when describing how to perform trimming/winsorization.)